src/Controllers/AccessTokenController.php

```
@@ -173,9 +173,10 @@ public function saveTeam(): void
     {
         $user   = $this->auth->user();
         $userId = (int) $user['id'];
+        $orgId  = (int) $user['org_id'];
         $team = trim((string) ($_POST['team'] ?? ''));
         $teamValue = $team !== '' ? $team : null;
-        User::update($this->db, $userId, ['team' => $teamValue]);
+        User::update($this->db, $userId, $orgId, ['team' => $teamValue]);
 // Verify the write actually landed
         $verify = $this->db->query('SELECT team FROM users WHERE id = :id LIMIT 1', [':id' => $userId])->fetch();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$verify'. (undefined)

(UnusedLocalVariable)